### PR TITLE
Feature shrimp app eva

### PR DESCRIPTION
**Pull request by hanoskoe**
added application shrimps
with new model Shrimp

### CODE REVIEW

File: src/shrimps/models.py [hunk 1]
```diff
@@ -21,7 +22,7 @@ class Shrimp(models.Model):
     ]
     name = models.CharField(max_length=126)
     # not now() because in that case the function would be executed straight away but we want to execute it at creation of the object
-    birth_date = models.DateTimeField('is born', default=timezone.now)
+    birth_date = models.DateTimeField('Birth Date', default=timezone.now)
     size = models.IntegerField()
     weight = models.IntegerField()
```
Comment (EliNovakova) [inline src/shrimps/models.py]: add units of weight (g) and size (mm)

**Comment by EliNovakova** [comment]
Hey Evi, maybe you should squash your migrations

**Comment by hanoskoe** [comment]
Thanks for feedback. Updated now. Please review :)

**Comment by EliNovakova** [comment]
It seems good to me :)

**Pull request merged**